ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -140,7 +135,7 @@ Item {
             height: parent.height
             visible: !root.rootStore.openCreateChat && isActiveChannel
             chatId: model.itemId
-            activeChatType: root.activeChatType
+            activeChatType: model.type
```

Comment:
We should probably rename that property then. It's implicitly the "active" chat type, because when the ChatContentView is visible, it's its own type that is active, but it could be confused as the type of the active chat, even if it's another chat.

Anyway, it can just be renamed `chatType`.

Reply:
@jrainville OK removed it completely :) We can get it from `chatContentModule.chatDetails.type` (some code was already doing that), so I just did alike